meta-lmp-base/recipes-kernel/linux/linux-lmp-dev.bb

```diff
@@ -12,7 +12,7 @@ python __anonymous() {
 FIO_LMP_GIT_URL ?= "github.com"
 FIO_LMP_GIT_NAMESPACE ?= "foundriesio/"
 
-LINUX_VERSION ?= "5.15-rc"
+LINUX_VERSION ?= "6.4.0-rc2"
```

Comment:
This change is fine, can be landed as a separated pr if needed.

Reply:
@ricardosalveti Ok, I separated it to PR https://github.com/foundriesio/meta-lmp/pull/1150